app/reports/subscription_collection_report.rb

```diff
@@ -0,0 +1,45 @@
+class SubscriptionCollectionReport
+  def initialize(subscriptions)
+    @subscriptions =
```

Comment:
I'm not sure if this is wrong according with the ruby style guide. But a better way and also I often see these cases on the project like:
```Ruby
@subscriptions = subscriptions.map do |subscription|
   SubscriptionReport.new(subscription, columns)
end

```